appveyor.yml

```diff
@@ -149,14 +151,23 @@ before_build:
 - echo %PATH%
 - if "%platform%"=="mingw-w64" set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
 - if "%platform%"=="mingw-w64" set PATH=C:\msys64\mingw64\bin;%PATH%
```

Comment:
The first line does string replacement. The second prepends a new entry to PATH.